src/MAUI/Maui.Samples/Samples/UtilityNetwork/TraceUtilityNetwork/readme.md

```diff
@@ -51,7 +51,7 @@ Tap on one or more features while 'Add starting locations' or 'Add barriers' is
 
 ## About the data
 
-The [Naperville electrical](https://sampleserver7.arcgisonline.com/server/rest/services/UtilityNetwork/NapervilleElectric/FeatureServer) network feature service contains a utility network used to run the subnetwork-based trace shown in this sample. Authentication is required and handled within the sample code.
+The [Naperville Electric Map](https://sampleserver7.arcgisonline.com/portal/home/item.html?id=be0e4637620a453584118107931f718b) web map contains a utility network used to run the subnetwork-based trace shown in this sample. Authentication is required and handled within the sample code.
```

Comment:
I think this reads a bit awkwardly "Naperville Electric Map web map contains"... maybe just "Naperville Electric Web Map contains"?